integration_test/ops_agent_test.go

```diff
@@ -3069,6 +3085,124 @@ metrics:
 	})
 }
 
+func getRecentServiceOutputForPlatform(platform string) string {
+	if gce.IsWindows(platform) {
+		cmd := strings.Join([]string{
+			"$Past = (Get-Date) - (New-TimeSpan -Minute 1)",
+			"Get-WinEvent -MaxEvents 10 -FilterHashtable @{ Logname='Application'; ProviderName='google-cloud-ops-agent'; StartTime=$Past } | select -ExpandProperty Message",
+		}, ";")
+		return cmd
+	}
+	return "sudo systemctl status google-cloud-ops-agent"
+}
+
+func listenToPortForPlatform(platform string) string {
+	if gce.IsWindows(platform) {
+		cmd := strings.Join([]string{
+			`echo '$Listener = [System.Net.Sockets.TcpListener]20202; $Listener.Start(); Start-Sleep -Seconds 600' > C:\script.ps1`,
```

Comment:
Are we sure  `C:\script.ps1` is always accessible? Maybe use /tmp or progdata?

Reply:
This https://github.com/GoogleCloudPlatform/ops-agent/pull/1051/commits/9af42e051cc3f8f3d97cc7f57cb43fe10c00763e avoids completely the issue of creating a script.